plugins/module_utils/CertACLHelper.cs

```diff
@@ -0,0 +1,282 @@
+using System;
+using System.ComponentModel;
+using System.Runtime.InteropServices;
+using System.Security.AccessControl;
+using System.Security.Cryptography.X509Certificates;
+using Microsoft.Win32.SafeHandles;
+using System.Security.Principal;
+
+
+//TypeAccelerator -Name Ansible.Windows.CertAclHelper.CryptHandle -TypeName CryptHandle
+//TypeAccelerator -Name Ansible.Windows.CertAclHelper.SafeSecurityDescriptorPtr -TypeName SafeSecurityDescriptorPtr
+//TypeAccelerator -Name Ansible.Windows.CertAclHelper.CertAccessRights -TypeName CertAccessRights
+//TypeAccelerator -Name Ansible.Windows.CertAclHelper.CertAccessRightsHelper -TypeName CertAccessRightsHelper
+//TypeAccelerator -Name Ansible.Windows.CertAclHelper.CertAclHelper -TypeName CertAclHelper
+
+namespace ansible_collections.ansible.windows.plugins.module_utils.CertACLHelper
+{
+
+    public class CryptHandle : SafeHandleZeroOrMinusOneIsInvalid
```

Comment:
I feel like a few of these classes should be `private` to avoid having outside users actually depend on this code without creating an actual public API.